packages/vuetify/src/components/VSelect/VSelect.ts

```diff
@@ -356,6 +356,9 @@ export default baseMixins.extend<options>().extend({
     getContent () {
       return this.$refs.menu && this.$refs.menu.$refs.content
     },
+    getIsInteractive () {
+      return this.isInteractive && !this.isDisabled && !this.$parent?.$props?.disabled
+    },
```

Comment:
I recommend implementing as a computed property instead for caching purposes: https://v2.vuejs.org/v2/guide/computed.html#Computed-Caching-vs-Methods